### PR TITLE
bug(python): Stop run when there's a call error

### PR DESCRIPTION
**Pull request by tebeka**
Fixes: ENG-1696

### CODE REVIEW

File: runtimes/pythonrt/runner/main.py [hunk 1]
```diff
@@ -194,14 +194,12 @@ def Start(self, request: pb.runner.StartRequest, context: grpc.ServicerContext):
 
     def Execute(self, request: pb.runner.ExecuteRequest, context: grpc.ServicerContext):
         with self.lock:
-            call = self.activity_calls.pop() if self.activity_calls else None
```
Comment (itayd) [inline runtimes/pythonrt/runner/main.py]: ah good i wanted to talk to you about that. if you do `pop` it will not work on retries.

**Review by itayd — COMMENTED** [review]
i see mostly logging changes?

File: runtimes/pythonrt/runner/main.py (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/runner/main.py]: Any action item for me?

**Comment by tebeka** [comment]
> i see mostly logging changes?

No. The main change is sending `Done` when there's an error so it'll stop the workflow.

File: runtimes/pythonrt/runner/main.py (hunk 1 above)
Comment (itayd) [reply runtimes/pythonrt/runner/main.py]: nope

**Review by itayd — APPROVED** [review]

**Pull request merged**